src/render/tutorials/getting-started-with-docker/2_0.html.md

```diff
@@ -33,120 +33,164 @@ Project has the following structure:
-    </div>
-  </div>
-  <div class="row">
-   <knotx:snippet data-knotx-task="bookslist">
-      {{#each books._result.items}}
-        <div class="col-sm-3">
-        <div class="card">
-          <img class="card-img-top"
-             src="{{volumeInfo.imageLinks.thumbnail}}"
-             alt="{{volumeInfo.title}}">
-          <div class="card-block">
-            <h4 class="card-title">{{volumeInfo.title}}</h4>
-            <p class="card-text">
-              {{#each volumeInfo.authors as |author|}}
-                {{author}}{{#unless @last}}, {{/unless}}
-              {{/each}}<br />
-              Published: {{volumeInfo.publishedDate}}
-            </p>
-          </div>
-         </div>
-      </div>
-      {{/each}}
-    </knotx:snippet>
-
-    </div>
-</div>
-</body>
-</html>
-```
-
-### Templates repository configuration
-We will use [fsRepoConnectorHandler](https://github.com/Knotx/knotx-repository-connector/tree/master/fs), its purposes are purely academical.
-It is not designed to be used as a production ready solution (you should use [httpRepoConnectorHandler](https://github.com/Knotx/knotx-repository-connector/tree/master/http) there).
-
-Create `knotx/conf/routes/handlers/fsRepoConnectorHandler.conf` file with the following:
-```hocon
-# Path to the directory on the local filesystem which will be the root for requested templates
-catalogue = "/usr/local/knotx/repository/"
-```
-This way we define the `repository` directory that will be our files content repository.
+## Create a new routing path
 
-### openapi.yml
-Open `knotx/conf/openapi.yml` and add following path definition:
+Let's take a look at `knotx/conf/openapi.yaml` file.
 
-```
-  /content/*:
+It's an [OpenAPI 3.0](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md) specification file that exposes APIs.
+
+> In OpenAPI terms, paths are endpoints (resources), such as /users or /reports/summary/, that your API exposes, and operations are the HTTP methods used to manipulate these paths, such as GET, POST or DELETE. [~source](https://swagger.io/docs/specification/paths-and-operations/))
+
+We can see three distinguishable parts of this file:
+
+ 1.  Information about our project, which is quite self-explanatory:
+ 
+      ```
+      ...
+      
+      openapi: "3.0.0"
+      info:
+        version: 1.0.0
+        title: Knot.x Starter Kit
+        description: Knotx Starter Kit
+        
+      ...
+      ```      
+ 2.  Server configuration that defines how our server behaves:
+ 
+      ```
+      ...
+      
+      servers:
+        - url: https://{domain}:{port}
+          description: The local API server
+          variables:
+            domain:
+              default: localhost
+              description: api domain
+            port:
+              enum:
+                - '8092'
+              default: '8092'
+      
+      ...
+      ```
+      Here we can see, for example, that our server will be registered on port `8092`.
```

Comment:
Some short note about variables could be worth writing. It may not be super obvious.

Reply:
done